osu.Game/Overlays/ChatOverlayV2.cs

```diff
@@ -346,5 +385,22 @@ private void handleChatMessage(string message)
             else
                 channelManager.PostMessage(message);
         }
+
+        private void cycleChannel()
+        {
+            List<Channel> overlayChannels = filterChannels(channelManager.JoinedChannels).ToList();
```

Comment:
This isn't working as expected. The order here is not the visual order and therefore results in weird traversal:

https://user-images.githubusercontent.com/191335/170233301-516997fe-649b-43a1-be07-868b392e7707.mp4

Also, the channel scroll view should be re-centering itself after selecting a new channel via keyboard. You can call `ScrollIntoView` on the drawable to make this happen.